apps/lifecycle.go

```diff
@@ -243,18 +243,14 @@ var _ = AppsDescribe("Application Lifecycle", func() {
 					return mem, disk, logs
 				}
 
-				It("for memory usage", func() {
+				It("for memory, disk and log usage", func() {
 					Eventually(func() float64 { m, _, _ := stats(); return m }, Config.CfPushTimeoutDuration()).Should(BeNumerically(">", 0.0))
-				})
-				Context("(cf-for-vms)", func() {
-					SkipOnK8s("App disk usage info unavailable")
```

Comment:
Please add the `SkipOnK8s` back. Some folks who still maintain cf-for-k8s have asked to keep k8s-related things in CATs for the time being.